yarn-project/aztec.js/src/account_manager/index.ts

```diff
@@ -92,6 +92,15 @@ export class AccountManager {
    */
   public async register(opts: WaitOpts = DefaultWaitOpts): Promise<AccountWalletWithPrivateKey> {
     const address = await this.#register();
+
+    await this.pxe.addContracts([
+      {
+        artifact: this.accountContract.getContractArtifact(),
+        completeAddress: address,
+        portalContract: EthAddress.ZERO,
+      },
+    ]);
```

Comment:
Not sure if this is the best place to put it but what this is trying to solve is: adding an existing account, that's been deployed through PXE A, to a new PXE, B and calling contracts using this new PXE instance, B.

Reply:
Makes sense! I doubt account contracts will have a corresponding portal, so I think this is ok.